packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -627,8 +627,11 @@ export function sendSignedTransaction<
 					};
 
 					try {
+						const { v , r , s, 
+								...txWithoutSigParams} = unSerializedTransactionWithFrom;
+						
 						await sendTxHelper.checkRevertBeforeSending(
-							unSerializedTransactionWithFrom as TransactionCall,
+							txWithoutSigParams as TransactionCall,
```

Comment:
No need to send v r s to `eth_call` as per EL Specs

https://github.com/ethereum/execution-apis/blob/584905270d8ad665718058060267061ecfd79ca5/src/eth/execute.yaml#L1

https://github.com/ethereum/execution-apis/blob/584905270d8ad665718058060267061ecfd79ca5/src/schemas/transaction.yaml#L274